pkg/security/security_profile/activity_tree/activity_tree.go

```diff
@@ -98,6 +100,17 @@ const (
 	MaxNodeGenerationType NodeGenerationType = 4
 )
 
+// SECLRuleOpts defines SECL rules options
+type SECLRuleOpts struct {
+	EnableKill bool
+	AllowList  bool
+	Lineage    bool
+	ImageName  string
+	ImageTag   string
+	Service    string
+	FIM        bool
+}
+
```

Comment:
could this struct and all new funcs around it could be in a separate file please ? :)